src/api/todos.ts

```diff
@@ -0,0 +1,20 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const USER_ID = 760;
+
+export const getTodos = () => {
+  return client.get<Todo[]>(`/todos?userId=${USER_ID}`);
+};
+
+export const createTodo = (todo: Partial<Todo>) => {
```

Comment:
`todo: Omit<Todo, "id">`
I believe you must have everything, but the id, right?